src/modules/utils/moment.js

```diff
@@ -13,6 +13,8 @@ import {
 	string,
 } from '@moderntribe/common/utils';
 
+export const TIME_FORMAT = 'h:mm a';
```

Comment:
Can we add a test for this one?

Reply:
There is a test for this, see https://github.com/moderntribe/tribe-common/pull/880/files#diff-1af408429616daceb23d7d1da2045fcdR31